pages/common/sleep.md

```diff
@@ -7,10 +7,14 @@
 
 `sleep {{seconds}}`
 
-- Delay in minutes:
+- Delay in [m]inutes. (Other units [d]ay, [h]our, [s]econd, [inf]inity can also be used):
 
 `sleep {{minutes}}m`
 
-- Delay in hours:
+- Pause for 1 day 3 hours:
 
-`sleep {{hours}}h`
+`sleep 1d 3h`
```

Comment:
```suggestion
`sleep 1d 3h`

- Delay for a specific amount of time ([m]inutes, [d]ay, [h]our, [s]econd, [inf]inity can be used):

`sleep {{time}}{{d|h|m|s|inf}}`
```

Reply:
If we keep the [changes on line 10](https://github.com/tldr-pages/tldr/pull/9631#discussion_r1051322748) this won't be required. Lets wait for that decision.